sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -26,6 +26,15 @@ select 5 div 0;
 select 5 div null;
 select null div 5;
 
+set spark.sql.legacy.integralDivide.returnBigint=true;
+
+select 5 div 2;
+select 5 div 0;
+select 5 div null;
+select null div 5;
+
+set spark.sql.legacy.integralDivide.returnBigint=false;
+
```

Comment:
Can we move these tests into the end of this file?

Reply:
I wanted to keep them close to the other `div` tests, but I am fine moving them. Let me know.

Reply:
Too many unrelated changes in the output file, so IMO it'd be better to do so. But, its ok to wait for other reviews.

Reply:
ok thanks, I see your point. cc @cloud-fan @dongjoon-hyun @viirya for their opinion on this. Thanks.

Reply:
Seems it's getting more complicated, can we create a new file for testing `div`?